consensus/types/src/chain_spec.rs

```diff
@@ -1038,4 +1039,63 @@ mod yaml_tests {
             .expect("should have applied spec");
         assert_eq!(new_spec, ChainSpec::minimal());
     }
+
+    #[test]
+    fn test_defaults() {
```

Comment:
And somewhere here verify that
```rust
assert_eq!(
    Uint256::from_dec_str(
        "115792089237316195423570985008687907853269984665640564039457584007913129638912",
    ), 
    Ok(default_terminal_total_difficulty)
);
```

that way the value does not need to be parsed in runtime